### PR TITLE
chore(release): v1.0.0-alpha.2

### PR DESCRIPTION
**Pull request by shadoath**
- [x] Replace `vNext` and `yyyy-mm-dd` with incremented version number and current date.
- [x] Add new `vNext` heading in CHANGELOG.
- [x] Increment synchronized package versions & peer-dependencies.
- [x] Commit with message `chore(release): <vNext>`
- [x] Tag commit: `git tag -a <vNext> -m "chore(release): <vNext>"`
- [x] Push branch and tag to origin: `git push && git push origin <vNext>`
- [x] Open PR
- [ ] _(after PR merge)_ Draft new release

### CODE REVIEW

**Review by slingner — APPROVED** [review]

**Comment by WilliamKelley** [comment]
Fixed title. 

Please rebase off main. had to revert and re-merge the previous PR for a fixed title and squash merge commit there as well.

Also, please copy the release checklist (see [here](https://github.com/prenda-school/design-system/pull/352))for confirmation.

File: libs/spark-icons/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...vNext) (yyyy-mm-dd)
+## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.2...vNext) (yyyy-mm-dd)
+
+## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...v1.0.0-alpha.2) (2022-02-25)
```
Comment (WilliamKelley) [inline libs/spark-icons/CHANGELOG.md:5]:
```suggestion
## [v1.0.0-alpha.2](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...v1.0.0-alpha.2) (2022-02-25)
```

File: libs/spark/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...vNext) (yyyy-mm-dd)
+## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.2...vNext) (yyyy-mm-dd)
+
+## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...v1.0.0-alpha.2) (2022-02-25)
```
Comment (WilliamKelley) [inline libs/spark/CHANGELOG.md:5]:
```suggestion
## [v1.0.0-alpha.2](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.1...v1.0.0-alpha.2) (2022-02-25)
```

**Review by WilliamKelley — CHANGES_REQUESTED** [review]

**Comment by WilliamKelley** [comment]
Oh and doing a rebase means you'll have to re-tag it (requires delete the current tag on local and origin).

**Review by WilliamKelley — APPROVED** [review]

**Pull request merged**